README.md

```diff
@@ -140,6 +140,21 @@ Returns group metadata for a given group:
 
 Lists all the authors (feedIds) who you know are part of the group with id `groupId`
 
+### `ssb.tribes.subtribes.create(groupId, opts, cb)`
+
+Mint a new private subgroup within a group.
```

Comment:
we need to say something to distinguish this from the `group.create`. I think we should say:

> A convenience method which :
> - mints a group
> - publishes a `link` in the parent group which advertises the existence of the subgroup
> - mints a dmKey for that group so that parent group member can send message to sub-group
> then creates a `link` inside an existing group to

Reply:
Cool :)